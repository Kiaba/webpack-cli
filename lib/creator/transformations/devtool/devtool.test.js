const defineTest = require('../../../transformations/defineTest');

defineTest(__dirname, 'devtool', 'devtools-0', {devtool : '\'source-map\''});
defineTest(__dirname, 'devtool', 'devtools-0', {devtool : '\'cheap-module-source-map\''});
defineTest(__dirname, 'devtool', 'devtools-0', {devtool : 'myVariable'});
defineTest(__dirname, 'devtool', 'devtools-0', {devtool : 'false'});
