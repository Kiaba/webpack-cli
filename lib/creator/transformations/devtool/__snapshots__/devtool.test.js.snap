// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`devtool transforms correctly using "devtools-0" data 1`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    devtool: 'source-map'
}
"
`;

exports[`devtool transforms correctly using "devtools-0" data 2`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    devtool: 'cheap-module-source-map'
}
"
`;

exports[`devtool transforms correctly using "devtools-0" data 3`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    devtool: myVariable
}
"
`;

exports[`devtool transforms correctly using "devtools-0" data 4`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    devtool: false
}
"
`;
